### PR TITLE
Display email address and phone number of a procuration proxy.

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

File: app/Resources/views/procuration_manager/request.html.twig [hunk 1]
```diff
@@ -59,7 +59,7 @@
                 {% if request.phone %}
                 <br />
                 Téléphone :
-                <a href="tel:{{ request.phone|phone_number_format(constant('\\libphonenumber\\PhoneNumberFormat::RFC3966')) }}" id="request-phone">
```
Comment (tgalopin) [inline app/Resources/views/procuration_manager/request.html.twig]: Why remove this?

Comment (hhamon) [inline app/Resources/views/procuration_manager/request.html.twig]: Because it generates it twice.

Comment (hhamon) [inline app/Resources/views/procuration_manager/request.html.twig]:
```
<a href="tel:tel:+33....">
```

Comment (hhamon) [inline app/Resources/views/procuration_manager/request.html.twig]: Maybe it's a browser bug...

Comment (hhamon) [inline app/Resources/views/procuration_manager/request.html.twig]: The `phone_number_format` filter already generates `tel:` prefix.

Comment (tgalopin) [inline app/Resources/views/procuration_manager/request.html.twig]: Are you sure of this? It's also used for display and I don't see the tel prefix

Comment (hhamon) [inline app/Resources/views/procuration_manager/request.html.twig]: capture d ecran 2017-04-11 a 13 47 00

File: app/Resources/views/procuration_manager/_proposals_list.html.twig [hunk 2]
```diff
@@ -4,10 +4,22 @@
...
             {% endif %}
+            <br/>
+            Email :
+            <a href="mailto:{{ proxy.emailAddress }}">
+                {{ proxy.emailAddress }}
+            </a>
+            {% if proxy.phone %}
+                <br />
+                Téléphone :
+                <a href="{{ proxy.phone|phone_number_format(constant('\\libphonenumber\\PhoneNumberFormat::RFC3966')) }}">
```
Comment (mykiwi) [inline app/Resources/views/procuration_manager/_proposals_list.html.twig]: what about create a global variable to store this constant and use it in everywhere ?

Comment (hhamon) [inline app/Resources/views/procuration_manager/_proposals_list.html.twig]: Make sense for another refactoring.

File: app/Resources/views/procuration_manager/request.html.twig (hunk 1 above)
Comment (tgalopin) [inline app/Resources/views/procuration_manager/request.html.twig]: Okay for me then

**Comment by tgalopin** [comment]
I think this is ready isn't it?

**Comment by hhamon** [comment]
Yes it is!

File: app/Resources/views/procuration_manager/_proposals_list.html.twig (hunk 2 above)
Comment (tgalopin) [inline app/Resources/views/procuration_manager/_proposals_list.html.twig]: Could you do a PR for this?

Comment (hhamon) [inline app/Resources/views/procuration_manager/_proposals_list.html.twig]: Yep tomorrow.

**Pull request merged**